pom.xml

```diff
@@ -107,6 +107,10 @@
             <version>${mockito-junit-jupiter.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
```

Comment:
Should we version this?

Reply:
Was thinking about it, but I guess we can always leave it pull the latest version until we run into problems

Reply:
I think we should always hard-version our dependencies or avoid running into problems later. Its safer for us

Reply:
I disagree only because it forces us to update our libraries as they will never get updated anyways. Additionally if anything does go wrong, we can simply start hard versioning.

Reply:
I disagree. If we don't hard-version, and it always pulls the latest, something will inevitably break at some point in the future, and we'll spend a long time figuring out what it was. Versioning is needed, always pulling the latest causes pain points (see: Sarama issues as one example).

Reply:
done